pkg/common/common.go

```diff
@@ -1,14 +1,54 @@
 package common
 
 import (
+	"errors"
+	"os"
+
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var RunLocal = (os.Getenv("env") == "runlocal")
```

Comment:
I'll change this to `ENV` and `local`, same as with the *dynatrace-sli-service*  and keptn core services.